client/src/lib/components/DatasetFormLayout/DatasetFormLayout.svelte

```diff
@@ -1,8 +1,56 @@
 <script>
```

Comment:
Oupsy c'était un oubli de ma part mais il faudrait rajouter ceci pour activer la prise en charge de TS dans ce fichier
```suggestion
<script lang="ts">
```

Reply:
ah oui mon linter est beaucoup plus bavard d'un coup :smile: